### ISSUE TITLE
EnderIO + Baubles issues

### ISSUE THREAD

**Issue by Cherry** [rwtema/extrautilities_old_issues#554]
Cross post https://github.com/SleepyTrousers/EnderIO/issues/2203

Any further insight or help would be appreciated.

**Comment by Cracks777** [comment]
Cherry, how good are you at java?

**Comment by Cherry** [comment]
Not at all. I'm a javascript (primarily node.js) guy, though I've dabbled in other languages throughout college and such.

**Comment by Cracks777** [comment]
well i ask because you reverted the comment on ender io, how to you build ender io from the git?

**Comment by Cherry** [comment]
The Forge wiki is a great source of information. Not the place in this issue though, feel free to message me directly if you like.

**Comment by Cracks777** [comment]
how do i message you directly?

**Comment by Cherry** [comment]
See the email address on my profile.

Closing this as per https://github.com/SleepyTrousers/EnderIO/pull/2213

**Comment by Cherry** [comment]
This surfaced again; https://github.com/SleepyTrousers/EnderIO/issues/2352
